### PR TITLE
Source Monday: timeout errors

### PR DESCRIPTION
**Pull request by davydov-d**
## What
This is a problem specific to Items stream, as it has additional rate limits
https://github.com/airbytehq/alpha-beta-issues/issues/61

## How
Added limit parameter, increased retry_factor

### CODE REVIEW

**Review by bazarnov — APPROVED** [review]

File: airbyte-integrations/connectors/source-monday/source_monday/spec.json [hunk 1]
```diff
@@ -31,8 +31,6 @@
               "auth_type": {
                 "type": "string",
                 "const": "oauth2.0",
-                "enum": ["oauth2.0"],
```
[the remark this answers is not in the document]
Comment (davydov-d) [reply airbyte-integrations/connectors/source-monday/source_monday/spec.json:34]: I believe it shouldn't break anything as we have a SAT asserting `enum` and `default` keywords are not used in `oneOf` structures

File: airbyte-integrations/connectors/source-monday/source_monday/source.py [hunk 2]
```diff
@@ -88,6 +91,13 @@ class Items(MondayStream):
     API Documentation: https://api.developer.monday.com/docs/items-queries
     """
 
+    limit = 100
+
+    @property
+    def retry_factor(self) -> int:
+        # this stream has additional rate limits, please see https://api.developer.monday.com/docs/items-queries#additional-rate-limit
+        return 30
```
[the remark this answers is not in the document]
Comment (davydov-d) [reply airbyte-integrations/connectors/source-monday/source_monday/source.py:99]: yep, but according to logs, from time to time the API allows us to make requests once in about 30 to 90 seconds.. that's weird but that's the fact, so I decided to set the `retry_factor` to a minimum value

[the remark this answers is not in the document]
Comment (davydov-d) [reply airbyte-integrations/connectors/source-monday/source_monday/source.py:99]: if this wont work, I'll increase it

**Pull request merged**